lightning/src/chain/chainmonitor.rs

```diff
@@ -262,6 +346,61 @@ where C::Target: chain::Filter,
 		self.monitors.write().unwrap().remove(funding_txo).unwrap().monitor
 	}
 
+	/// Indicates the persistence of a [`ChannelMonitor`] has completed after
+	/// [`ChannelMonitorUpdateErr::TemporaryFailure`] was returned from an update operation.
+	///
+	/// Thus, the anticipated use is, at a high level:
+	///  1) This [`ChainMonitor`] calls [`Persist::update_persisted_channel`] which stores the
+	///     update to disk and begins updating any remote (e.g. watchtower/backup) copies,
+	///     returning [`ChannelMonitorUpdateErr::TemporaryFailure`],
+	///  2) once all remote copies are updated, you call this function with the
+	///     `completed_update_id` that completed, and once all pending updates have completed the
+	///     channel will be re-enabled.
+	///
+	/// Returns an [`APIError::APIMisuseError`] if `funding_txo` does not match any currently
+	/// registered [`ChannelMonitor`]s.
+	pub fn channel_monitor_updated(&self, funding_txo: OutPoint, completed_update_id: MonitorUpdateId) -> Result<(), APIError> {
```

Comment:
Do we need special handling for the `MonitorUpdateId` being `CLOSED_CHANNEL_UPDATE_ID`?

Reply:
Hmmm, I don't think so? We're not exactly in a rush to return a monitor-updated event if the channel is closed (it won't do anything no matter what). And the monitor updates from channelmanager don't get in the way of returning events back to the channelmanager, so I think that's fine?